src/main/java/com/authlete/common/api/AuthleteApi.java

```diff
@@ -265,7 +266,28 @@ public interface AuthleteApi
      *
      * @since 2.29
      */
-    TokenListResponse getTokenList() throws AuthleteApiException;
+    public TokenListResponse getTokenList() throws AuthleteApiException;
```

Comment:
In Java conventions, `public` is not prepended to methods in an interface. Please remove `public`.